Sources/NIO/NIOAny.swift

```diff
@@ -255,6 +255,22 @@ public struct NIOAny {
             return o
         }
     }
+    
+    /// Force-unwraps the wrapped message as a given type `T`
+    ///
+    /// returns: The wrapped `T`, or crashes if the wrapped message is not a `T`
+    @inlinable
+    public func unwrap<T>(as type: T.Type = T.self) -> T {
```

Comment:
If we're okay adding public API for unwrapping I think it would be better for it to return `T?` (in which case it might just be better to `tryAs` public). I think this will help to discourage people from using it over `unwrap{In,Out}boundIn` in handlers. 

If we do have a force unwrap API as well then I think it should be obvious from the name that it can blow up.

Reply:
You make a good point. If we go down that route we can even just mark the existing functions as `public` and the job is done. Hoping to spin up a discussion to see if the general consensus is for/against this.

Reply:
I misread what you first said, I didn't mean to just repeat what you said back at you haha!